ci-operator/step-registry/hypershift/mce/multi-version-test/trigger-jobs/hypershift-mce-multi-version-test-trigger-jobs-commands.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+declare -A guest_to_job_aws=(
+    [4.14]="periodic-ci-openshift-hypershift-release-4.14-periodics-mce-e2e-aws-critical"
+    [4.15]="periodic-ci-openshift-hypershift-release-4.15-periodics-mce-e2e-aws-critical"
+    [4.16]="periodic-ci-openshift-hypershift-release-4.16-periodics-mce-e2e-aws-critical"
+    [4.17]="periodic-ci-openshift-hypershift-release-4.17-periodics-mce-e2e-aws-critical"
+    [4.18]="periodic-ci-openshift-hypershift-release-4.18-periodics-mce-e2e-aws-critical"
+)
+declare -A mce_to_guest=(
+    [2.4]="4.14"
```

Comment:
I don't think it hurts to test 4.14 and 4.15 but AWS GA'd with 4.16 support.

Reply:
Since the GA version of Agent KubeVirt is 4.14, I’ve configured it this way for now to ensure support for all platforms. Of course, I’ll optimize it.